### DISCUSSION TITLE
JPEG XL animations

### DISCUSSION THREAD

**Discussion by OronDF343** [dlemstra/Magick.NET#1040] [Help]
Is it currently possible to support JPEG XL animations?
If I save to a .jxl file it only saves the first frame.
If I save it with MagickFormat.Jxl it creates a file that seems to be the correct size, but if I read it again it only loads 1 frame. All viewer applications only display 1 frame, and djxl (official decoder) fails to decompress it.

**Comment by dlemstra** [comment]
Jpeg XL is still in early development and we currently only support reading a single image. Reading and writing multiple frames is something that we will probably add in the future.
